runtime/mono-runtime.m.t4

```diff
@@ -130,6 +130,16 @@ MONO_API <#= export.ReturnType #>
 <# } #>
 	return <#= export.EntryPoint #>_func (<#= export.ArgumentNames #>);
 }
+<# } #>
+
+<# foreach (var export in exports) {
+	if (!export.Optional)
+		continue; #>
+bool
+<#= export.EntryPoint #>_exists ()
+{
+	return <#= export.EntryPoint #>_func != NULL;
+}
```

Comment:
Inlining it would require some header changes (mono-runtime.h) to add the `_func` variables to the headers, and I don't think it's worth it because it only applies to XM apps using system mono.

Also these `_exists` methods are not used in generated code, only in `runtime.m`.